src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -51,28 +51,25 @@
         oPaymill.bOkExpiry = paymill.validateExpiry(oPaymill.sExpiryMonth, oPaymill.sExpiryYear);
         oPaymill.bOkCvv = paymill.validateCvc(oPaymill.nCvv);
 
-
-
         if (oPaymill.bOkNumber !== true) {
             var oLiNumber = document.createElement('li');
-            oLiNumber.innerText = 'Card number invalid';
+            oLiNumber.innerHTML = 'Card number invalid';
```

Comment:
You don't need to use ``innerHTML`` here because your content is not an HTML content